cmake/dependencies.cmake

```diff
@@ -138,12 +119,11 @@ rapids_find_package(nlohmann_json REQUIRED
 
 # prometheus
 # =========
-set(PROMETHEUS_CPP_VERSION "1.0.0" CACHE STRING "Version of Prometheus-cpp to use")
-include(deps/Configure_prometheus)
+morpheus_utils_configure_prometheus_cpp(${PROMETHEUS_CPP_VERSION})
 
 # libcudacxx
 # =========
-include(deps/Configure_libcudacxx)
+morpheus_utils_configure_libcudacxx(${LIBCUDACXX_VERSION})
```

Comment:
Why are these two different and pass variables into the functions? We should have them all be the same.